Tasks/UniversalPackagesV0/universalpublish.ts

```diff
@@ -149,6 +149,12 @@ export async function run(artifactToolPath: string): Promise<void> {
             retries++; 
         }
 
+        if(publishStatus === packageAlreadyExistsError){
```

Comment:
```suggestion
        if(publishStatus === packageAlreadyExistsError && versionRadio === "custom"){
```

Reply:
and we can remove `versionRadio !== "custom"` on line 120

Reply:
moving this before the while will not cover all cases. In the case of non-custom version, after 1 retry(maybe uncommon, but possible) no error will be thrown if the package was not published

Reply:
I see, agree.